hw/ip/pwrmgr/data/pwrmgr.hjson

```diff
@@ -119,6 +119,15 @@
     },
   ],
 
+  countermeasures: [
+    { name: "BUS.INTEGRITY",
+      desc: "End-to-end bus integrity scheme."
+    }
+    { name: "FSM.SPARSE",
```

Comment:
don't take this too seriously yet, i'm going to come back and update the description more.
This PR is more to do the actual conversion